object/provider.go

```diff
@@ -59,12 +59,22 @@ type Provider struct {
 	OutputPricePerThousandTokens float64 `xorm:"DECIMAL(10, 4)" json:"outputPricePerThousandTokens"`
 	Currency                     string  `xorm:"varchar(100)" json:"currency"`
 
-	Network     string `xorm:"varchar(100)" json:"network"`
-	Chain       string `xorm:"varchar(100)" json:"chain"`
-	TestContent string `xorm:"varchar(100)" json:"testContent"`
-	IsDefault   bool   `json:"isDefault"`
-	State       string `xorm:"varchar(100)" json:"state"`
-	BrowserUrl  string `xorm:"varchar(200)" json:"browserUrl"`
+	OrgId          string `xorm:"varchar(100)" json:"orgId"`
```

Comment:
Re-use existing fields

Some fields like "contractName", "contractMethod" can be hard-coded for now